tests/integration/test_main.py

```diff
@@ -370,3 +370,46 @@ def test_action__workflow_run__post_comment(
     assert result == 0
     assert get_logs("INFO", "Comment file found in artifact, posting to PR")
     assert get_logs("INFO", "Comment posted in PR")
+
+
+def test_action__pull_request__comment_template(
```

Comment:
When a fixture is only relevant for one specific test, I prefer putting the fixture next to the test.
Also, I'm not sure why we're using a fixture here ? Wouldn't it be clearer to define the `COMMENT_TEMPLATE` in the test directly ?

Finally... This is a high level intergation test. Wouldn't it be easier to implement this test as a lower-level unit test (in https://github.com/ewjoachim/python-coverage-comment-action/blob/v2/tests/unit/test_template.py )

Reply:
I originally thought it would be clearer to implement all configurations as a fixture. But I absolutely understand that it's not worth it for a one-time use. I have fixed that.

The way I see it, testing if the `get_markdown_comment` function can handle templates is already implemented in `test_template`. But this test should test the `clean_comment_template` function, which was also correctly shown as not covered before. But I didn't see any other way than to implement that high-level, because that is only called when `COMMENT_TEMPLATE` is set in the configuration. How would I implement this as a low-level unittest?

Reply:
I could, of course, write a test that simply calls `Config.from_environ` once with and once without `COMMENT_TEMPLATE` and then checks to see if the correct value was stored. Is that what you mean?

Reply:
Done